app/scoring/page.tsx

```diff
@@ -0,0 +1,9 @@
+export default function Scoring() {
+    var currentSelectedContestant = "Jacob"
```

Comment:
I didn't know you were a contestant. Congratulations

Reply:
Hehe, in the fantasy league I am 😁